vector-im/element-android
# [Device management] Sign out a session (PSG-742)

## Type of change


## Content


Adding a signout button in the session overview screen.

## Motivation and context


Closes #7190 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/192982503-c79201ca-9844-4eda-8494-b1565a2e8ef7.gif" width=300 />

## Tests



- Enable the feature flag for new device management
- Go to Settings -> Security & Privacy -> Show all sessions (WIP, V2)
- Press a session item to go to the corresponding session overview screen
- Press the signout button
- Check reauth is asked if needed
- Check the screen is exited after signout is done
- Check the session has disappeared from the screens
- Try the same process with the current session
- Check the app is restarted in this case

## Tested devices

- OS version(s): Android 11

## Checklist
